libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -168,6 +168,10 @@ impl Connector {
         }
     }
 
+    pub fn list_connections(&self) -> Result<CmResponse, ConnectionManagerError> {
```

Comment:
Why doesn't this return `Vec<String>` and translate any errors of the response into `ConnectionManagerError`.  That way `CmResponse` remains internal

Reply:
updated